### ISSUE TITLE
nodejs platform is not supported

### ISSUE THREAD

**Issue by artygus** [microsoft/Oryx#288]
This morning deployment started to fail with the error `nodejs platform is not supported`. MS support team was unable to resolve the issue and suggested filling bug report on github 

> Command: oryx build /home/site/wwwroot -o /home/site/wwwroot -l nodejs --language-version 8.1 -i /tmp/8d70b6da48128e5 -p compress_node_modules=tar-gz
> Build orchestrated by Microsoft Oryx, https://github.com/Microsoft/Oryx
> You can report issues at https://github.com/Microsoft/Oryx/issues
> 
> Oryx Version       : 0.2.20190521.9, Commit: c2ee5d8bcba3b398c8390fbe2939d888dbf94352
> Build Operation ID : |3XEBGdkeqic=.9113fed2_
> Repository Commit  : bc8303a4806bd0d58068b8009c7c27d257141e76
> 
> 'nodejs' platform is not supported. Supported platforms are: nodejs, python, dotnet, php
> 'nodejs' platform is not supported. Supported platforms are: nodejs, python, dotnet, php\n/opt/Kudu/Scripts/starter.sh oryx build /home/site/wwwroot -o /home/site/wwwroot -l nodejs --language-version 8.1 -i /tmp/8d70b6da48128e5 -p compress_node_modules=tar-gz

**Comment by gpcastro** [comment]
Thanks for reaching out. Is it possible that environment variable/app setting `DISABLE_NODEJS_BUILD` is set to `true` in your web app?

**Comment by gpcastro** [comment]
we interpret it as if you don't want your node app to be built and skip it; can you change its value to `true` or remove it?
Here's the list of variables we interpret: https://github.com/microsoft/Oryx/blob/master/doc/configuration.md

**Comment by artygus** [comment]
Aha, I see. I deploy prebuilt code and only need a `serve` package installed to serve static content, no build actions required.

Problem I encountered today, maybe not related to the oryx itself, is that I use sub-directory for node project (i.e. package.js, package-lock.json file located in sub-directory), which I specify as PROJECT config variable, but it stopped working.

Moving out node project from sub-directory to root fixed the issue and my app went live, but deployment was always marked as failed. After I removed DISABLE_NODEJS_BUILD variable build was finally marked as successful, thanks!

I think the error message is very confusing, could it be replaced with something more meaningful?
